uber/tests/models/test_badge_funcs.py

```diff
@@ -15,10 +15,11 @@ def session(request):
 
 @pytest.fixture(autouse=True)
 def teardown_range_check(request):
-    def _check_range():
-        with Session() as session:
-            check_ranges(session)
-    request.addfinalizer(_check_range)
```

Comment:
From looking through your tests, I'd expect ``teardown_range_check`` to succeed, since in the two unit tests you added, we're not creating any holes in the badge numbers.  We still want to turn this check off when ``c.SHIFT_CUSTOM_BADGES`` is off, because we plan to write more tests in the future which WILL create those kinds of holes.  Still, it doesn't surprise me that it worked for you.